scenario/fixtures/chaincode/golang/echo/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/hyperledger/fabric-gateway/scenario/echo/go
+
+go 1.15
+
+require (
+	github.com/hyperledger/fabric-contract-api-go v1.1.1
+	github.com/pkg/errors v0.9.1
```

Comment:
As per previous comments, I'm not sure it's a good idea to use this package when the core libraries give us similar functionality